crawlers/crawlers.interfaces.ts

```diff
@@ -17,3 +17,8 @@ export interface ICrawlerModule {
   start: (args: ICrawlerModuleConstructorArgs) => Promise<void>;
   config?: ICrawlerModuleConfig;
 }
+
+export interface ICollectionSchemaInfo {
```

Comment:
What does mean 'info'?

Reply:
`Info` here means 'some structure about collection schema'). Because it's not single collection schema, but object with `collectionId` and `schema` field.